### PR TITLE
ci: Run windows tests.

### PR DESCRIPTION
**Pull request by ceyusa**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -42,7 +42,9 @@ jobs:
 
       - name: Install
         run:
-          DESTDIR=$REPO_NAME-release ninja -C builddir install
+          ninja -C builddir install
+        env:
+          DESTDIR: ${{ env.REPO_NAME }}-release
```
Comment (dabrain34) [inline .github/workflows/main.yml:47]: This was not working properly when I was trying

**Comment by ceyusa** [comment]
it looks like sourceforge has screwed pkgconfiglite through choco

**Comment by dabrain34** [comment]
> it looks like sourceforge has screwed pkgconfiglite through choco

https://github.com/chocolatey/choco/issues/2866

**Comment by dabrain34** [comment]
This MR is not relevant anymore as most of the patches have been applied

**Pull request closed without merging**